libafl_frida/src/asan_rt.rs

```diff
@@ -864,91 +229,1477 @@ impl AsanRuntime {
             };
             assert!(mapping.unwrap() as usize == max_start);
         }
-        (max_start, end)
+        (max_start, end)
+    }
+
+    /// Determine the tls start, end for the currently running thread
+    fn current_tls() -> (usize, usize) {
+        let tls_address = unsafe { tls_ptr() } as usize;
+
+        #[cfg(target_os = "android")]
+        let tls_address = tls_address & 0xffffffffffffff;
```

Comment:
`MAX`?

Reply:
It's not max... it's 7 bytes.

Reply:
Ow. :)
I guess don't try this on 32 bit